switchsai/CMakeLists.txt

```diff
@@ -3,20 +3,38 @@
 # Copyright 2022 Intel Corporation
 # SPDX-License-Identifier: Apache 2.0
 
-add_library(switchsai_o OBJECT
-   sai.c
-   saifdb.c
-   saiinternal.h
-   sailag.c
-   saineighbor.c
-   sainexthop.c
-   sainexthopgroup.c
-   sairoute.c
-   sairouterinterface.c
-   saitunnel.c
-   saiutils.c
-   saivirtualrouter.c
-)
+if(DPDK_TARGET)
```

Comment:
I was thinking, instead of changing switchsai (which we claim to be common across ES2K and DPDK), can we add switch_lag.c, switch_pd_lag.h and switch_pd_lag.c files to switchapi/dpdk/CMakeLists.txt

and stub out PD code in switch_pd_lag.c with return value SWITCH_STATUS_NOT_IMPLEMENTED. 

requesting @vsureshkumarp / @ffoulkes opinion on these both approaches.